BlogPetNews.API/Program.cs

```diff
@@ -44,3 +44,5 @@
 app.AddNewsEndpoints();
 
 app.Run();
+
+public partial class Program { }
```

Comment:
o que esta linha 48 esta fazendo?

Reply:
Como criei um factory para conseguir mudar os services para os fakes, preciso acessar o program. Mas como ele tem um nível de proteção, não consigo acessar ele se não deixar ele publico e da erro na compilação se não coloca isso:

![image](https://github.com/jheanr/blogNoticias/assets/50773932/34f2539c-2241-44cc-b8cb-69c8ce5473a3)

Reply:
Para o .Net 6 precisa fazer isso mesmo na Program. [Documentação](https://learn.microsoft.com/en-us/aspnet/core/test/integration-tests?view=aspnetcore-6.0#basic-tests-with-the-default-webapplicationfactory).